### PR TITLE
Bt upgrade

### PR DESCRIPTION
**Pull request by ch4rr0**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
Can you revert the last commit and keep the recipes in it's subdirs ... this might be useful for people that are interested in maintaining them.

**Comment by ch4rr0** [comment]
done

File: recipes/bowtie/meta.yaml [hunk 1]
```diff
@@ -4,28 +4,37 @@ about:
...
+
+source:
+  fn: bowtie-1.2-source.zip
+  url: http://downloads.sourceforge.net/project/bowtie-bio/bowtie/1.2.0/bowtie-1.2-source.zip
+  sha256: b1052de4253007890f6436e6361d40148bc2a5a9dd01827bb9f34097747e65f8
 
 build:
-  number: 2
-  skip: False
+  number: 1
```
Comment (bgruening) [inline recipes/bowtie/meta.yaml]: This can start with 0.

File: recipes/bowtie2/meta.yaml [hunk 2]
```diff
@@ -5,31 +5,36 @@ about:
...
-  md5: c3d5dd807f053b9fd7cc786312e6e3fb # [linux]
-  fn: bowtie2-2.2.8-macos-x86_64.zip # [osx]
-  url: http://downloads.sourceforge.net/project/bowtie-bio/bowtie2/2.2.8/bowtie2-2.2.8-macos-x86_64.zip # [osx]
-  sha256: f1ed9079d5699eb123e74a749df447820422b7c8216d69f2ffb3f3487844a1b7 # [osx]
+  fn: bowtie2-2.3.0-source.zip
+  url: http://downloads.sourceforge.net/project/bowtie-bio/bowtie2/2.3.0/bowtie2-2.3.0-source.zip
+  sha256: f9f841e780e78b1ae24b17981e2469e6d5add90ec22ef563af23ae2dd5ca003c
 
 build:
   number: 2
```
Comment (bgruening) [inline recipes/bowtie2/meta.yaml]: please start with 0

**Comment by bgruening** [comment]
Great package @ch4rr0! Welcome and thanks for your work on bowtie! :)

**Comment by ch4rr0** [comment]
How can I get the following pages updated to reflect these changes?
http://bioconda.github.io/recipes/bowtie/README.html
http://bioconda.github.io/recipes/bowtie2/README.html

**Comment by daler** [comment]
These pages are from an automatic docs build, and it appears we're having issues with it. I'll take a look.

**Comment by daler** [comment]
The issue is that @ch4rr0 docs are generated in another repo (bioconda-utils) when there are updates to the master branch of that repo. No changes there in a couple of days hence no docs regenerated. I'll bump it now.

@bgruening @johanneskoester should we set up a cron job on travis to build nightly docs?

**Comment by bgruening** [comment]
Yeah, I just did that! Cron is now running every 24h. Thanks for looking into it.

**Comment by daler** [comment]
Nice, thanks!

**Comment by ch4rr0** [comment]
Thank you! I do see that the pages have been updated.

**Pull request merged**